tests/integration/test_spans.py

```diff
@@ -454,7 +475,9 @@ def test_span_ingestion(
         headers={"Content-Type": "application/x-protobuf"},
     )
 
+    print("Waiting for spans..")
     spans = list(spans_consumer.get_spans(timeout=10.0, max_attempts=6))
+    print("Done waiting for spans.")
```

Comment:
```suggestion
    spans = list(spans_consumer.get_spans(timeout=10.0, max_attempts=6))
```
Please also delete the `print`s below (these should be cough by [automation](https://github.com/getsentry/relay/issues/3398) 😞)